@hesburgh-wse/gatsby-theme-marble/src/components/Internal/UserAnnotation/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import typy from 'typy'
+import queryString from 'query-string'
+import CalloutBox from 'components/Internal/CalloutBox'
+import Attribution from 'components/Internal/Attribution'
+import Link from 'components/Internal/Link'
+import UserCartouche from 'components/Internal/UserCartouche'
+const UserAnnotation = ({ location }) => {
+  const qs = queryString.parse(location.search)
+  if (qs.a) {
+    // fetch annotations
+    const annotations = getAnnotationsFromIds(qs.a.split(','))
+    return (
+      <React.Fragment>
+        {
+          typy(annotations).safeArray.map((annotation, index) => {
+            return (
+              <CalloutBox key={index}>
```

Comment:
Does annotation have a unique id that can be used as the key instead of index?

Reply:
It will when it has real (or more realistic) data behind it. I have a lot of questions about the shape of this data, so this is just a placeholder until the data questions are resolved.